content/dstu2/binary.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Binary | DSTU 2 API
+---
+
+# Binary
+
+* TOC
+{:toc}
+
+## Retrieve by id
+
+List an individual Binary by its id:
+
+    GET /Binary/:id
+
+### Implementation Notes
```

Comment:
In other resources, the implementation notes aren't a heading.  I'd change this to one italic line of "Implementation Notes - Binary for DiagnosticReport"